sdk/src/main/assets/hcaptcha-form.html

```diff
@@ -104,20 +123,9 @@
     function onHcaptchaLoaded() {
         try {
             var renderConfig = getRenderConfig();
-            var rqdata = bridgeConfig.rqdata;
-            var hCaptchaID = hcaptcha.render('hcaptcha-container', renderConfig);
+            hCaptchaID = hcaptcha.render('hcaptcha-container', renderConfig);
 
-            if (rqdata) {
-                hcaptcha.setData(hCaptchaID, {
-                    rqdata: rqdata
-                });
-            }
-
-            if (renderConfig.size === 'invisible') {
-                hcaptcha.execute(hCaptchaID);
-            } else {
-                BridgeObject.onLoaded();
```

Comment:
`BridgeObject.onLoaded()` is never called w/ current changes.

Reply:
Good catch, fixed and retested, invisible, compact and normal sizes